roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -36,8 +46,11 @@
     "get_network:provider:physical_network": "rule:admin_only",
     "get_network:provider:segmentation_id": "rule:admin_only",
     "get_network:queue_id": "rule:admin_only",
+    "get_network_ip_availabilities": "rule:admin_only",
+    "get_network_ip_availability": "rule:admin_only",
```

Comment:
I think we may want to open this one up to cloud_admin, so that they can see how much floating IPs and such they have used and available. This has been an asked for feature.